### ISSUE TITLE
Custom color tables causing crashes/warnings

### ISSUE THREAD

**Issue by sawellons** [agurvich/FIRE_studio#17]
I am having some issues with the function load_my_custom_color_tables() in firestudio/utils/stellar_utils/colors.py.  The first time I tried to make a stellar image the code crashed with the following error:
```
----> 1 firestudio.utils.stellar_utils.colors.load_my_custom_color_tables()

~/home/python/firestudio/utils/stellar_utils/colors.py in load_my_custom_color_tables()
     98                        (1.0, 0.0, 0.0))}
     99     my_cmap = matplotlib.colors.LinearSegmentedColormap(fna,cdict_tmp,256);
--> 100     matplotlib.cm.register_cmap(name=fna,cmap=my_cmap);
    101 
    102 

/srv/conda/envs/notebook/lib/python3.7/site-packages/matplotlib/cm.py in register_cmap(name, cmap, override_builtin)
    147         if not override_builtin and name in __builtin_cmaps:
    148             msg = f"Trying to re-register the builtin cmap {name!r}."
--> 149             raise ValueError(msg)
    150         else:
    151             msg = f"Trying to register the cmap {name!r} which already exists."

ValueError: Trying to re-register the builtin cmap 'rainbow'.
```

(Note: this is not the original error, this is a reproduction from me calling the function myself.)

It is upset about trying to create a colormap with the same name as one of the built-in matplotlib maps.  I was able to sidestep this by renaming the “rainbow” table to “rainbow_custom”.

A more minor issue is that each subsequent time I make a stellar image, I get a series of warnings about re-registering a name for a map that already exists:
`~/home/python/firestudio/utils/stellar_utils/colors.py:28: UserWarning: Trying to register the cmap 'heat_red' which already exists.`
(but for every map defined in that function).  This does not break anything, but it is somewhat annoying.

**Comment by agurvich** [comment]
I see. Strange that this has never come up before, I haven't added a call to this particular function anywhere ¯\_(ツ)_/¯

**Comment by sawellons** [comment]
I agree, strange!  If I find myself in a situation where I get a deeper stack trace I'll let you know.  Maybe it's from a matplotlib update or something.
